packages/env/lib/wordpress.js

```diff
@@ -73,14 +73,16 @@ async function configureWordPress( environment, config ) {
 
 	const port = environment === 'development' ? config.port : config.testsPort;
 
+	const wpUrl = `localhost:${ port }`;
+
 	// Install WordPress.
 	await dockerCompose.run(
 		environment === 'development' ? 'cli' : 'tests-cli',
 		[
 			'wp',
 			'core',
 			'install',
-			`--url=localhost:${ port }`,
+			`--url=${ wpUrl }`,
 			`--title=${ config.name }`,
 			'--admin_user=admin',
```

Comment:
`config.config.WP_TESTS_DOMAIN` should just default to `localhost:${ port }` in `config.js` then.

Reply:
I think the URL should vary based on the environment, and we don't have a way to differentiate based on that in the config (yet). So for now, I'm not sure there is a better place

Though, I don't know if there is a scenario in which `WP_TESTS_DOMAIN` should _not_ point at the test container. But it seems weird to set it to be the same url on both instances. Maybe you want to point a test runner at your main dev instance.

Reply:
You are still defaulting it, but in an ad-hoc way where it's used.

I think it should be done in `config.js` where all of this is handled.

Reply:
Ok. I’ll just hardcode “localhost:${testsPort}” for the tests URL all the time now and we can handle the dev environment case once we have support for different configuration for each environment

Reply:
It's not much cleaner, but it works for now 🤷. IMO it might still make sense to revisit this in the future. Not sure if there is ever a scenario where the tests URL should be different from the instance URL? Not very familiar with that.